modules/gdscript/gdscript_editor.cpp

```diff
@@ -777,7 +777,22 @@ static void _find_annotation_arguments(const GDScriptParser::AnnotationNode *p_a
 	}
 }
 
+static void _find_built_in_variants(Map<String, ScriptLanguage::CodeCompletionOption> &r_result, bool only_types = false) {
+	for (int i = 0; i < Variant::VARIANT_MAX; i++) {
+		if (Variant::Type(i) == Variant::Type::NIL) {
+			if (only_types) continue;
```

Comment:
```suggestion
			if (only_types) {
				continue;
			}
```

Reply:
Though since the previous `if` condition would only happen once, you could group them together (and maybe rename `only_types` to `exclude_nil`?):
```
if (!exclude_nil && Variant::Type(i) == Variant::Type::NIL) {
```